specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/execution.json

```diff
@@ -0,0 +1,1103 @@
+          "example": "MySourceDatastore"
+        },
+        "path": {
+          "description": "The path to the source directory or file.\r\nThe path should not include the blob container\r\nname or fileshare name.\r\nIf not set, the path defaults to the root\r\nof the file share or blob container.\r\nThe cloud file or the contents of cloud directory will be copied to the working directory of the run\r\nbefore the user code runs.",
+          "type": "string",
+          "example": "source/projects/projectAbc/"
+        }
+      }
+    },
+    "DataReferenceConfiguration": {
+      "description": "A class for managing DataReferenceConfiguration.",
+      "type": "object",
+      "properties": {
+        "dataStoreName": {
+          "description": "The name of the data store.",
+          "type": "string",
+          "example": "myblobstore"
+        },
+        "mode": {
+          "description": "Operation on the datastore, mount, download, upload.",
+          "enum": [
+            "Mount",
+            "Download",
+            "Upload"
+          ],
+          "type": "string",
+          "example": "Mount",
+          "x-ms-enum": {
+            "name": "DataStoreMode",
+            "modelAsString": true
+          }
+        },
+        "pathOnDataStore": {
+          "description": "Relative path on the datastore.",
+          "type": "string",
+          "example": "/images/validation"
+        },
+        "pathOnCompute": {
+          "description": "The path on the compute target.",
+          "type": "string"
+        },
+        "overwrite": {
+          "description": "Whether to overwrite the data if existing.",
+          "type": "boolean",
+          "example": "False"
+        }
+      }
+    },
+    "EnvironmentDefinition": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the environment.",
+          "type": "string",
+          "example": "mydevenvironment"
+        },
+        "version": {
+          "description": "The environment version.",
+          "type": "string",
+          "example": "1"
+        },
+        "python": {
+          "$ref": "#/definitions/PythonSection",
+          "description": "Settings for a Python environment."
+        },
+        "environmentVariables": {
+          "description": "Definition of environment variables to be defined in the environment.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "docker": {
+          "$ref": "#/definitions/DockerSection",
+          "description": "The definition of a Docker container."
+        },
+        "spark": {
+          "$ref": "#/definitions/SparkSection",
+          "description": "The configuration for a Spark environment."
+        },
+        "inferencingStackVersion": {
+          "description": "The Inferencing stack version.",
+          "type": "string",
+          "example": "latest"
```

Comment:
How do I get the list of valid versions?

Reply:
done